src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/render/nametags/ModuleNametags.kt

```diff
@@ -46,12 +46,21 @@ object ModuleNametags : Module("Nametags", Category.RENDER) {
 
     object Health : ToggleableConfigurable(this, "Health", true) {
         val fromScoreboard by boolean("FromScoreboard", false)
+
+    }
+
+    object HealthColor : ToggleableConfigurable(this, "HealthColor", true) {
+        val greenHealth by int("Green", 12, -1..40)
+        val yellowHealth by int("Yellow", 8, -1..40)
+        val redHealth by int("Red", -1, -1..40)
```

Comment:
At first I thought this was to select the color, but now I can see it's to select the threshold. I think this is overkill and does not need to be configurable.

Reply:
So do you think custom colors should be allowed?